lib/actions/add.js

```diff
@@ -17,52 +17,31 @@ const Boom = require('boom');
 
 module.exports = function addToCollection(route, options) {
 
-    return (request, reply) => {
+    return async (request, h) => {
```

Comment:
the moment I see a single letter variable name in a place I'm unfamiliar, I feel lost. just a personal note, not really relevant to this, as I'm sure 'h' is normal here.

Reply:
I agree, and don't really care for the h, but it's idiomatic hapi, so...